packages/adapters/wagmi/src/connectors/AuthConnector.ts

```diff
@@ -105,9 +105,8 @@ export function authConnector(parameters: AuthParameters) {
 
     async isAuthorized() {
       const provider = await this.getProvider()
-      const { isConnected } = await provider.isConnected()
 
-      return isConnected
+      return Promise.resolve(provider.getLoginEmailUsed())
```

Comment:
We can use the local storage value to see if someone has connected before. Since `isAuthorized` function checks whether the connector has connected previously and is still authorized.

This means when wagmi is trying to reconnect if `isAuthorized` returns false it'll ignore the connection ([here](https://github.com/wevm/wagmi/blob/4825ef4a901c7281e1ca69d2f3378d167108c98a/packages/core/src/actions/reconnect.ts#L77-L78) is the code from wagmi)